md/api-extensions.md

```diff
@@ -52,8 +52,10 @@ A REST API extension example resource and API extension viewer page are provided
 
 To use the examples, define a [custom permissions](rest-api-authorization.md#custom-permissions-mapping) mapping for `demoPermission`:
 
-* Edit `$BONITA_HOME/client/tenants/1/conf/custom-permissions-mapping.properties`.
+* Retrieve platform configuration using the [setup tool](BonitaBPM_platform_setup.md#update-platform-conf).
+* Edit `$PLATFORM_CONF/current/tenants/1/tenant_portal/custom-permissions-mapping.properties`.
```

Comment:
Ok, after comparing with other pages `platform_conf/current/tenants/1/...` is enough.